tests/acceptance/26_cf-net/serial/cf-net_connect.cf

```diff
@@ -0,0 +1,47 @@
+body contain mypowershell
+{
+    !windows::
+        useshell => "useshell";
+    windows::
+        useshell => "powershell";
+}
+
+body common control
+{
+    inputs => {
+        "../default.cf.sub",
+        "../run_with_server.cf.sub"
+    };
+    bundlesequence => { default("$(this.promise_filename)") };
+}
+
+bundle agent init
+{
+    methods:
+        "any" usebundle => dcs_fini("$(G.testdir)/cf-net_connect.txt");
+
+        "any" usebundle => generate_key;
+        "any" usebundle => trust_key;
+
+        "any" usebundle => start_server("$(this.promise_dirname)/cf-net_connect.srv.cf");
+}
+
+bundle agent test
+{
+    commands:
+        "$(sys.cf-net)"
+            args => " -H 127.0.0.1:15000 connect > $(G.testdir)/cf-net_connect.txt",
+            contain => mypowershell;
+}
+
+bundle agent check
+{
+    methods:
+        "" usebundle => dcs_passif_output("Connected & authenticated successfully to '127.0.0.1:15000'", "", "cat /tmp/cf-net.txt", $(this.promise_filename));
```

Comment:
Instead of catting the file and checking the output I might just use regline to search the file for the content and define the class.

classes: "pass" expression => regline("Connected ........", $(g.testdir)/file);

methods:
  pass:: dcs_pass($(this.promise_filename));
  !pass:: dcs_fail($(this.promise_filename));